pygmt/src/makecpt.py

```diff
@@ -120,10 +120,12 @@ def makecpt(**kwargs):
         level :func:`pygmt.makecpt` is called.
     reverse : str
         Set this to ``True`` or **c** [Default] to reverse the sense of color
-        progression in the master CPT. Set this to **z** to reverse the sign
-        of z-values in the color table. Note that this change of z-direction
-        happens before ``truncate`` and ``series`` values are used so the
-        latter must be compatible with the changed z-range. See also
+        progression in the master CPT. Also the foreground and background colors
+        are exchanged, including those specified by :gmt-term:`COLOR_BACKGROUND` and
+        :gmt-term:`COLOR_FOREGROUND`. Set this to **z** to reverse the sign of z-values
+        in the color table. Note that this change of z-direction happens before
```

Comment:
```suggestion
        in the color table. Note that this change in z-direction happens before
```

Reply:
Hm. I think I prefer to keep "of" because the z-direction itself is changed, and not something "in" z-direction.
Maybe adding a "the" makes it sound nicer?

```suggestion
        in the color table. Note that this change of the z-direction happens before
```